app/scripts/components/common/page-header/logo-container/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ComponentType } from 'react';
+import { Tip } from '../../tip';
+import { LinkProperties } from '$types/veda';
+import './styles.scss';
+
+/**
+ * LogoContainer that is meant to integrate in the default
+ * page header without the dependencies of the veda virtual modules
+ * and expects the Logo SVG to be passed in as a prop - this will
+ * support the instance for refactor
+ */
+
+export default function LogoContainer({
+  linkProperties,
+  Logo,
+  title,
+  version,
+}: {
+  linkProperties: LinkProperties;
+  Logo?: JSX.Element;
+  title: string;
+  subTitle?: string;
+  version?: string;
+}) {
+  const LinkElement: ComponentType<any> =
+    linkProperties.LinkElement as ComponentType<any>;
+
+  return (
+    <div id='logo-container'>
+      <LinkElement id='logo-container-link' {...{ [linkProperties.pathAttributeKeyName]: '/' }}>
+        {Logo}
+        <span>{title}</span>
+      </LinkElement>
+      {version && (
```

Comment:
Do we want to keep it that way? Display a beta tag when there is a version specified?